Classes/CodingDojo.sc

```diff
@@ -1,19 +1,21 @@
 CodingDojo {
-	var <username, <password, <serveraddress, <serverport;
+	var <username, <password, <serveraddress, <serverport, <groupName, <groupPassword;
 	var <oscrouter, <syncText;
 	var <>turnTime = 300, <remainTime, <timer;
 	var <pilot, <copilot, <nextCopilot, <myStatus, <order;
 	var <win, <uv;
 
-	*new { arg username, password, serveraddress = "bgo.la", serverport = 55555;
-		^super.newCopyArgs(username.asSymbol, password.asSymbol, serveraddress.asString, serverport).init;
+	*new { arg username, password, serveraddress = "bgo.la", serverport = 55555,
+		groupName, groupPassword;
+		^super.newCopyArgs(username.asSymbol, password.asSymbol, serveraddress.asString, serverport, groupName, groupPassword).init;
 	}
```

Comment:
if not making making separate vars in CodingDojo, pass group name and password into init